iml-manager-cli/src/stratagem.rs

```diff
@@ -98,12 +98,14 @@ pub struct StratagemRemoveData {
 pub struct StratagemScanData {
     /// The name of the filesystem to scan
     filesystem: String,
-    /// The report duration
-    #[structopt(short = "r", long = "report", parse(try_from_str = parse_duration))]
-    report_duration: Option<u64>,
-    /// The purge duration
-    #[structopt(short = "p", long = "purge", parse(try_from_str = parse_duration))]
-    purge_duration: Option<u64>,
+    /// The report duration, specified as a humantime string
+    /// EX: 1hour
+    #[structopt(short = "r", long = "report", min_values = 1)]
+    report_duration: Option<Vec<String>>,
```

Comment:
Does it need to be optional with min_values = 1?

Reply:
probably semantics, but no report duration has a different meaning than the flag with nothing there.